client/account-recovery/lost-password/lost-password-form/style.scss

```diff
@@ -0,0 +1,22 @@
+.lost-password-form__title {
+	font-size: 1.5em;
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/8955/commits/0067de58d62d5c49aa7e125b67b67e83d8863aa8